.github/workflows/slash-command-dispatch.yml

```
@@ -21,5 +21,6 @@ jobs:
           token: ${{ steps.generate-token.outputs.token }}
           commands: |
             format
+            test-gmt-master
```

```suggestion
            test-gmt-latest
```
What do you think about changing the name to 'latest' to match the Github Actions name?

---

When we say "the latest GMT", we usually mean "the latest stable GMT release" (i.e., 6.1.1). 

What about "GMT Dev Tests" and `/test-gmt-dev`?

---

Yep, dev sounds good and is shorter too.

---

Done in 6df8e340cc9462efec6365ccdbe9f8c644da3fea and 72b7dfae6207e1461ae4a51295efbe1c2d1722ef